package.json

```diff
@@ -57,7 +57,8 @@
     "package": "npm run build:fast && npm run build:standalone && npm run build:installer",
     "release": "ts-node --transpile-only automation/run.ts release",
     "pretest": "npm run build",
-    "test": "npm run test:source && npm run test:standalone",
+    "test": "npm test:shrinkwrap && npm run test:source && npm run test:standalone",
+    "test:shrinkwrap": "npm i && npm dedupe && npm i && npm run catch-uncommitted",
```

Comment:
I've added this which should help to keep junk out of the shrinkwrap by having the CI check

Reply:
But `npm run catch-uncommitted` specifically ignores `npm-shrinkwrap.json` -- I don't remember why, maybe you do:
* `"catch-uncommitted": "ts-node --transpile-only automation/run.ts catch-uncommitted"`
in `package.json`
* `npx catch-uncommitted --catch-no-git --skip-node-versionbot-changes --ignore-space-at-eol`
at https://github.com/balena-io/balena-cli/blob/v12.12.1/automation/build-bin.ts#L310-L314
* `":(exclude)npm-shrinkwrap.json"`
https://github.com/balena-io-modules/catch-uncommitted/blob/v1.5.0/catch-uncommitted.sh#L16

Reply:
Oh, I think the reason why `catch-uncommitted` ignores the `npm-shrinkwrap.json` file is related to the fact that the file includes the CLI version number, which versionbot automatically updates alongside the Changelog file. Then again, the change is committed to repo, or is it? If it was committed, then `catch-uncommitted` should be OK with it...

Reply:
Reworked this so it should now work